internal/db/schema/migrations/oss/postgres/60/01_ldap.up.sql

```diff
@@ -398,10 +466,13 @@ select
   am.discover_dn,
   am.anon_group_search,
   am.upn_domain,
+  am.enable_groups,
+  am.use_token_groups,
   -- the string_agg(..) column will be null if there are no associated value objects
   string_agg(distinct url.url, '|') as urls,
   string_agg(distinct cert.certificate, '|') as certs,
-
+  string_agg(distinct concat_ws('=', aam.from_attribute, aam.to_attribute), '|') as account_attribute_map,
```

Comment:
Could this be a generated column instead?

Reply:
I'm not following.

Reply:
Nevermind, I missed that this is an aggregation, we could have a [generated column](https://www.postgresql.org/docs/current/ddl-generated-columns.html) for the `concat_ws('=', aam.from_attribute, aam.to_attribute)` per row but we'd still have to aggregate it. Also introducing generated columns would require Postgres 12 and I think we still support 11?

Reply:
I guess it could be generated if we stopped supporting 11, but with that said: we need a view for the aggregation anywho, so creating a generated column doesn't seem to buy us anything.